data-roadmap/data_roadmap/data_set_description_schema.yaml

```diff
@@ -2,23 +2,26 @@
 name: str(required=True)
 source: str(required=True)
 relevance_to_environmental_justice: str(required=False)
-data_formats: enum('GeoJSON', 'Esri Shapefile (SHP, DBF, SHX)', 'GML', 'KML/KMZ',
-  'GPX', 'CSV/XLSX', 'GDB', 'MBTILES', 'LAS', required=True)
-spatial_resolution: enum('State/territory', 'County', 'Zip code', 'Census tract',
-  'Census block group', 'Exact address or lat/long', 'Other', required=True)
-public_status: enum('Not Released', 'Public', 'Public for certain audiences', 'Other',
-  required=True)
-sponsor: str(required=True)
+data_formats:
+  enum('GeoJSON', 'Esri Shapefile (SHP, DBF, SHX)', 'GML', 'KML/KMZ',
```

Comment:
Nice cleanup!

Reply:
We need to add data formats to this, eg TXT. Maybe this shouldn't be an enum yet.